modules/ossm-jaeger-config-elasticsearch.adoc

```diff
@@ -0,0 +1,43 @@
+////
+This TASK module included in the following assemblies:
+-ossm-configuring-jaeger.adoc
+////
+
+[id="ossm-jaeger-config-elasticsearch_{context}"]
+= Configuring the Elasticsearch index cleaner job
+
+When the {ProductShortName} Operator creates the `ServiceMeshControlPlane` it also creates the custom resource definition (CRD) for Jaeger.  The Jaeger operator then uses this CRD when creating Jaeger instances.
+
+When using Elasticsearch storage, by default a job is created to clean old traces from it.  To configure the options for this job, you edit the Jaeger custom resource (CR) are listed below so you can customize it for your use case.
+
+[source,yaml]
+----
+  apiVersion: jaegertracing.io/v1
+  kind: Jaeger
+  spec:
+    strategy: production
+    storage:
+      type: elasticsearch
+      esIndexCleaner:
+        enabled: false
+        numberOfDays: 7
+        schedule: "55 23 * * *"
+
+----
+
+.Elasticsearch index cleaner parameters
+|===
+|Parameter |Values |Description
+
+|enabled
+|true/ false
+|Enable or disable the index cleaner job.
+
+|numberOfDays
+|integer value
+|Number of days to wait before deleting a record.
```

Comment:
record -> index
The job cleans (deletes) indexes - where the indexes contain traces for a day (generally)